### DISCUSSION TITLE
Can we mask pixels to be rendered?

### DISCUSSION THREAD

**Discussion by osylum** [mitsuba-renderer/mitsuba3#654] [Q&A] [answered]
Hi,

I saw that there is a crop option to render only part of the film. As well, from https://mitsuba.readthedocs.io/en/latest/src/rendering/scripting_renderer.html, it would be possible to create custom rays that are emitted for selected pixels. How can I select pixels to be emitted while re-using existing sensor and integrator?

Thank you

**Comment by njroussel** [comment, marked as answer]
Hi @osylum 

This is currently not possible. You would need to write your own custom sensor. It should just be a matter of copying an exisiting one and adding a mask that is exposed.

**Comment by osylum** [comment]
ok, I'll do that then, thank you
